### ISSUE TITLE
Try to answer the question of "Dream and happiness" 

### ISSUE THREAD

**Issue by LillyWu** [LillyWu/LikeWater#3]
I have been lost for a while: "How should I balance the happiness and the dream you have!"

Based on my previous life experience, achieving a goal requires great effort, and most of the time, it tends to be not very happy. 

If happiness is the ultimate goal of life, how can we achieve a big dream?

**Comment by phamquiluan** [comment]
I am here waiting for the answer :smile:

**Comment by LillyWu** [comment]
Probably I can answer this question now. 

I would say there is no strong correlations between dream and happiness.
What matters is how you can make your dream come true in a happy way.

Anxiety, stress, unhappiness, or other negative emotions occur anyway. But what we can do is learn how to manage these and think in a positive way.  Sometimes, we just need to get through it with time, and keep going. 

On the way to your dream, I think it is the best part of living a life. 
It requires some soft skills, like how to handle the negative emotions, difficulties, and uncertainties, etc. 
In a nutshell, to be a strong and resilient people, keeping learning to solve problems,  it helps we walk farther than planned.
